velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +491,85 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, weekDate) {
+  const auto weekDate = [&](const StringView& date) {
+    auto week =
+        evaluateOnce<int64_t>("week(c0)", std::make_optional(parseDate(date)))
+            .value();
+    auto weekOfYear =
+        evaluateOnce<int64_t>(
+            "week_of_year(c0)", std::make_optional(parseDate(date)))
+            .value();
+    VELOX_CHECK(
```

Comment:
Can we use EXPECT_EQ or VELOX_CHECK_EQ to get a better error message, i.e. have the values being compared be included in the message?

Reply:
Changed to VELOX_CHECK_EQ, thanks for the suggestion!